guides/release/components/template-lifecycle-dom-and-modifiers.md

```diff
@@ -321,14 +321,23 @@ Let's start with the HTML we're working with:
 <button type="button">Pause</button>
 ```
 
-Next, let's add an event handler to the `Play` button:
+It might be tempting to try to handle this by managing the whole set of interactions at the component level. But there are three reasons we might want to tackle this using a modifier instead:
 
-```handlebars {data-filename="app/components/audio-player.hbs" data-diff="-3,+4"}
+1. Components don't have access to DOM elements directly. We'd have to render the page, push an element back up into the component class, and *then* wire it up. This would come with performance costs, because we would have to render twice before anything could work. It would also 
```

Comment:
This trails off oddly:

> It would also

Reply:
Heh, I turned this into the next bullet point and forgot to remove it. Thank you!